tools/king/king.xml

```diff
@@ -0,0 +1,160 @@
+<tool id="king" name="KING" version="2.2.4">
+    <requirements>
+	<requirement type="package" version="2.2.4">king</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    king -b '$genotype' --fam '$family' --bim '$map' $related $duplicate $kinship $ibdseg $ibs $homog
+
+    #if $degree
+    --degree $degree
+    #end if
+
+    #if $projection
+    --projection $projection
+    #end if
+
+    $unrelated $build $cluster $rplot
+    > $kingoutput
+    ]]></command>
+    <inputs>
+	<param name="genotype" type="data" format="bed" help="Binary genotype File" />
```

Comment:
the format of this file is not `bed` but actually a PLINK variant. See here:

https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/datatypes_conf.xml.sample#L445

so the format should actually be `pbed`.